extensions/interactions/CodeRepl/CodeReplRulesServiceSpec.js

```diff
@@ -148,6 +148,27 @@ describe('Code REPL rules service', function() {
     });
   });
 
+  describe('\'output contains\' rule', function() {
+      var RULE_INPUT = {
+        x: '1'
+      };
+
+      it('should check if output contains some content', function() {
```

Comment:
Consider also adding some tests that ensure that normalization of spaces, newlines, etc. in the rule input and code output works as expected.